vespalib/src/tests/process/process_test.cpp

```diff
@@ -0,0 +1,112 @@
+// Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+#include <vespa/vespalib/process/process.h>
+#include <vespa/vespalib/data/slime/slime.h>
+#include <vespa/vespalib/data/slime/json_format.h>
+#include <vespa/vespalib/data/simple_buffer.h>
+#include <vespa/vespalib/gtest/gtest.h>
+
+using vespalib::Input;
+using vespalib::Output;
+using vespalib::Process;
+using vespalib::SimpleBuffer;
+using vespalib::Slime;
+using vespalib::slime::JsonFormat;
+
+//-----------------------------------------------------------------------------
+
+TEST(ProcessTest, simple_run_ignore_output) {
+    EXPECT_TRUE(Process::run("echo foo"));
+}
+
+TEST(ProcessTest, simple_run_ignore_output_failure) {
+    EXPECT_FALSE(Process::run("false"));
+}
+
+//-----------------------------------------------------------------------------
+
+TEST(ProcessTest, simple_run) {
+    vespalib::string out;
+    EXPECT_TRUE(Process::run("echo -n foo", out));
```

Comment:
Consider `/bin/echo` instead of `echo`. The `sh` builtin `echo` does not understand `-n` option on darwin.

Reply:
fixed